extensions/xla/generate/adapter.py

```diff
@@ -40,6 +41,7 @@ def setup(
     Args:
         prompt: The prompt/instruction (Alpaca style).
         input: Optional input (Alpaca style).
+        sys_prompt: Optional system prompt.
```

Comment:
can we pls add some more explanation on what it does of how it could be used?

Reply:
In the adapter I basically followed the shortened description style of the other arguments.